_posts/2021-05-16-overview-of-architecture-review-processes.md

```diff
@@ -0,0 +1,253 @@
+---
+layout: post
+date:   2021-05-29 09:02:08 +0100
+title: "Overview of different Architecture Review Processes"
+categories:
+  - Architecture
+  - Organization
+  - Architecture Review Process
+published: true
+---
+Recently, I have been asked about what software architecture review processes exist in order to better steer architecture.
+An architecture review aims at different goals such as finding software design issues early in the development before they get costly.
+Architecture review processes, for example, formalize different steps.
```

Comment:
I do not think so, because the first sentence is about architecture review, the second is about architecture review processes. You can also perform an isolated architecture review without a formalized process such as a one-time architecture review from an external company. However, establishing a process is something different.